baselines/compute/src/v1/addresses_client.ts.baseline

```diff
@@ -369,23 +369,23 @@ export class AddressesClient {
       'project': request.project || '',
     });
     this.initialize();
-    return this.innerApiCalls.delete(request, options, callback);
+    return this.innerApiCalls.unaryDelete(request, options, callback);
```

Comment:
same here, let's keep `innerApiCalls` name just `delete`. We only want to rename the user-visible method, without touching any internals.